### 1. Вероятность того, что стрелок попадет в мишень, выстрелив один раз, равна 0.8. Стрелок выстрелил 100 раз. Найдите вероятность того, что стрелок попадет в цель ровно 85 раз.

In [1]:
# Применяем формулу Барнулли
import math  
C = (math.factorial(100)/(math.factorial(85)*math.factorial(15)))
p = 0.8**85
q = 0.2**15
print(f'Искомая вероятность: {C*p*q}')



Искомая вероятность: 0.048061793700746556


Ответ: Искомая вероятность: 0.048061793700746556

_______________________________________________________________________________________________________________________________

### 2. Вероятность того, что лампочка перегорит в течение первого дня эксплуатации, равна 0.0004. В жилом комплексе после ремонта в один день включили 5000 новых лампочек. Какова вероятность, что ни одна из них не перегорит в первый день? Какова вероятность, что перегорят ровно две?

In [2]:
# Применим формулу Пуассона 
p=0.0004
n=5000
m=0
m2=2
print(f'Вероятность что ни одна лампочка не перегорит в первый день: {(p*n)**m/math.factorial(m)*math.e**(-(p*n))}')
print(f'Вероятность что 2 лампочки перегорят в первый день: {(p*n)**m2/math.factorial(m2)*math.e**(-(p*n))}')


Вероятность что ни одна лампочка не перегорит в первый день: 0.1353352832366127
Вероятность что 2 лампочки перегорят в первый день: 0.2706705664732254


In [ ]:
Ответ: Вероятность что ни одна лампочка не перегорит в первый день: 0.1353352832366127
Вероятность что 2 лампочки перегорят в первый день: 0.2706705664732254

_______________________________________________________________________________________________________________________________

### 3. Монету подбросили 144 раза. Какова вероятность, что орел выпадет ровно 70 раз?

In [3]:
# Применяем формулу Барнулли
C = (math.factorial(144)/(math.factorial(74)*math.factorial(70)))
p = 0.5**74
q = 0.5**70
print(f'Искомая вероятность: {C*p*q}')

Искомая вероятность: 0.06281178035144776


Ответ: Искомая вероятность: 0.048061793700746556

_______________________________________________________________________________________________________________________________

### 4. В первом ящике находится 10 мячей, из которых 7 - белые. Во втором ящике - 11 мячей, из которых 9 белых. Из каждого ящика вытаскивают случайным образом по два мяча. Какова вероятность того, что все мячи белые? Какова вероятность того, что ровно два мяча белые? Какова вероятность того, что хотя бы один мяч белый?

In [4]:
print(f'Вероятность достать два белых мяча из первой корзины 7/10*6/9, \n' 
      f'Вероятность достать два белых мяча из второй корзины 9/11*8/10.\n'
      f'Вероятность того, что все мячи белые: {7/10*6/9*9/11*8/10}')

Вероятность достать два белых мяча из первой корзины 7/10*6/9, 
Вероятность достать два белых мяча из второй корзины 9/11*8/10.
Вероятность того, что все мячи белые: 0.3054545454545454


In [10]:
print(f'Вероятность достать два белых мяча из первой корзины и 2 не белых из второй 3/10*2/9*2/11*1/10, \n' 
      f'Вероятность достать один белый мяч из первой корзины и один из второй 3/10*3/9*9/11*2/10.\n'
      f'Вероятность достать два не белых мяча из первой корзины и два белых из второй 3/10*2/9*9/11*8/10.\n'
      f'Вероятность того, что ровно два мяча белые: {3/10*2/9*2/11*1/10+3/10*3/9*9/11*2/10+3/10*2/9*9/11*8/10} \n')

Вероятность достать два белых мяча из первой корзины и 2 не белых из второй 3/10*2/9*2/11*1/10, 
Вероятность достать один белый мяч из первой корзины и один из второй 3/10*3/9*9/11*2/10.
Вероятность достать два не белых мяча из первой корзины и два белых из второй 3/10*2/9*9/11*8/10.
Вероятность того, что ровно два мяча белые: 0.061212121212121204 



In [8]:
# Найдём вероятность что все вытянутые мячи не белые
print(f'Вероятность достать два не белых мяча из первой корзины 3/10*2/9, \n' 
      f'Вероятность достать два не белых мяча из второй корзины 2/11*1/10.\n'
      f'Вероятность того, что все мячи не белые: {3/10*2/9*2/11*1/10} \n'
      f'Вероятность того, что хотя бы один мяч белый: {1-3/10*2/9*2/11*1/10}')

Вероятность достать два не белых мяча из первой корзины 3/10*2/9, 
Вероятность достать два не белых мяча из второй корзины 2/11*1/10.
Вероятность того, что все мячи не белые: 0.0012121212121212121 
Вероятность того, что хотя бы один мяч белый: 0.9987878787878788


Ответ:
Вероятность того, что все мячи белые: 0.3054545454545454

Вероятность того, что ровно два мяча белые: 0.061212121212121204

Вероятность того, что хотя бы один мяч белый: 0.9987878787878788